### DISCUSSION TITLE
治癒系聊天機器人Woebot

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#298] [General]
在科技的世代，人們不是太忙，就是一心多用，沒時間深入理解他人的想法和感受。過去我們會個別且面對面地跟親朋好友聊近況，現在則多半透過簡訊、推特或社群媒體聯絡交流，我們漸漸都忘記了「聆聽」的重要性，寂寞在社會蔓延。
當自己感到難過卻不知道該找誰抒發情緒，或者只是無聊想找個人聊天的時候，因應科技社會產生的治癒系聊天機器人似乎就是個不錯的選擇。
[數位心理治療](https://www.bnext.com.tw/article/65013/ai-psychologist-therapy)

**Comment by chun1203** [comment]
這樣的機器人感覺對無助又不知道和誰傾訴的時候非常有幫助

**Comment by HSU-YU-JEN** [comment]
我覺得這個想法很棒，在之前大部分的人都覺得AI沒有辦法心理健康做些什麼，但事實上他們可以!!

**Comment by MAKERRRofKEVIN** [comment]
感謝同學分享，個人之前有用過名為Mitsuku的聊天機器人，這只是英國人發明的，當時好像有通過四次圖靈測驗，拿來練英文還滿方便的，比較可惜的是現在已經沒有在開放用戶使用了

**Comment by 107602505** [comment]
透過AI去學習心理治療師的洞察力與同理心，開發Woebot聊天機器人除了能幫助偏遠地區的患者，或許也可以幫助哪些不擅向他人敞開心扉的族群，感覺未來會有很大的市場。

**Comment by EricAgumonJou** [comment]
Woebot聊天機器人感覺也可以用於醫院或診所上，讓病人的身心靈感受到安撫和關愛～
或許小朋友以後生病了就會迫不及待想去醫院看Woebot聊天機器人（大誤＃＃）。

**Comment by EricAgumonJou** [comment]
Woebot聊天機器感覺未來可以結合長照機器人或應用於養老產業上，為長者提供生活上更多的協助和照顧。

**Comment by 20010906** [comment]
這類型的聊天機器人感覺會很受歡迎，當人心情低落時，非常需要傾訴的對象以及關心，希望之後能有機會體驗

**Comment by 109802503gloria** [comment]
我覺得也許是因為使用Woebot時，知道自己是在跟一個機器人聊天，而不是一個真人，人們能更容易放下心防說心事，而機器人還帶有理性的句子能幫助自己脫離情緒，冷靜梳理，很適合想一個人靜靜時使用。

**Comment by cody910120** [comment]
我怕機器人的回應太官方QQ我還是比較喜歡跟朋友喝酒屁話啦，畢竟比較真性情

**Comment by mesakiyuta** [comment]
感謝分享，Woebot在心理治療上或許還不能發揮跟心理治療師的效果，但做為過渡，我認為非常的好，且知道自己是跟機器人聊天，就不用擔心負面情緒影響到他人，文章中也有寫出她會分析的問題，對梳理自己的想法感覺也很有幫助。

**Comment by ChinChun0220** [comment]
謝謝分享，我會把這個推薦給我的朋友… 需要坦承自己需要聆聽也是很難，這種可以在夜晚需要的時候自己跟機器人聊天

**Comment by neo910831** [comment]
身邊感覺還真的有朋友需要這個，能發揮半個心理治療師的效果也好，至少比我這個外行人還有用

**Comment by YuHengChen54** [comment]
這個聊天機器人的回答都好和善、好正向，真的可以調適生活中不愉快的心情

**Comment by dabbie109303501** [comment]
跟機器人說心事會比跟人說話來的好，帶來的心理負擔比較小，也不用擔心會遭到批評

**Comment by Zoe06060606** [comment]
這種機器人感覺能減輕現代人的厭世程度
而且也能讓人毫無負擔的宣洩自己的情緒
除了資安的問題會讓人有點擔心

**Comment by wenchi9023** [comment]
在不想要將自己的負面情緒丟到他人身上的情況下，Woebot確實是個很好的抒發窗口!
感謝同學分享機器人也有溫暖、激勵人心的一面

**Comment by bonny123865** [comment]
我覺得Woebot看起來是一個還不錯的聊天機器人!
尤其現在可能很多人單身+外住，沒什麼機會或是不想打擾朋友
這種聊天機器人就剛好能讓自己抒發一點情緒，雖然說改善的效果可能因人而異，但是被關懷的感覺真的很棒!!

**Comment by Pietra1226** [comment]
對於現在少子化且大家都比較享受一個人生活的型態開始出現，這種機器人不管是對於老人或是自己住的年輕人都是很好的選擇。

**Comment by ching020721** [comment]
聊天機器人對於我這種有時候不想要打擾朋友，又需要抒發情緒與事情時，是很好的選擇。
雖然成效可能因人而異，畢竟對於AI大家還是不會認定它是人類，聊天起來肯定跟人類有差距，並認定其為「既定程序」
但我還是希望這項開發能夠持續發展進步，畢竟也是有人會需要的，說不定哪天我們也都會用到！

**Comment by evanchou1204** [comment]
治癒系機器人好暖心，現在真的在哪裡都壓力很大，好需要被機器人治癒。

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
現今社會 大家的生活壓力越來越大 生活部調也相當緊湊 很少能找個人好好的聊聊天 是個不錯的發明

**Comment by CPCpyt** [comment]
當今人們社會的生活越來越繁雜快速，
取得心靈上的慰藉真的是很重要的課題。

**Comment by ruthchen1020** [comment]
我覺得如果可以當成一種情緒宣洩的管道很好，但是不要過度依賴，我覺得人類不應該寄託感情在機器人上面。

**Comment by StevenWanglolz** [comment]
我只需要這個機器人、一隻狗、一隻貓就滿足了

**Comment by 202283730032** [comment]
请问国内现在怎么使用这款woebot呢

**Comment by larry40305** [comment]
有些心裡的話可能不適合跟周遭的人說，或許這類的治癒系聊天機器人是個不錯的選擇

**Comment by 202283730032** [comment]
谢谢。

**Comment by Lipunpun** [comment]
感謝分享，這個新興的科技，或許能夠在這個生活壓力極大的生活，帶來一片溫暖，或許能夠作為自閉症患者與社會接軌的一個過度期。

**Comment by Jack-1D** [comment]
Woebot在心理治療方面可能真的有成效。但我認為人類有些微小的情感與感受是機器所不能理解的，臨床上有時候也無法獲取類似的資料。

**Comment by Lipunpun** [comment]
真的覺得很需要這個軟體，很多自閉症的人都已經不想和人接觸了，接觸只會讓病情惡化，我認為這種方式可以讓他們找到自己與外界社會的一種相處模式。

**Comment by zhiron544** [comment]
我覺得Woebot是一個很棒的創新，它可以讓我們在感到孤獨或困擾的時候，有一個可以傾訴的對象。我想它也可以用在學校裡，幫助學生減壓和增加自信。

**Comment by 202283730032** [comment]
谢谢。
